tests/framework/ROM/TimeSeries/ARMA/Segmented/ercotNC_all_hourly_2007.csv

```diff
@@ -0,0 +1,8761 @@
+Time,Speed,GHI,Temperature,Demand
```

Comment:
questions:
1) Do we need such huge file for testing?
2) In case we do, are we authorize to use this one?
3) In case we do, do we have this file already in the repo for other tests (we can abvoid to readd it)?

Reply:
1. Yes, anything less is statistically meaningless and impossible to segment meaningfully.
2. Yes, it's openly available to anyone with access to the world-wide web.
3. No, this is not repeated anywhere, and other ARMA tests have had drastically-reduced histories to work on.

Reply:
ok